src/deps/c_ares.zig

```diff
@@ -11,6 +11,81 @@ pub const ares_socket_t = c_int;
 pub const ares_sock_state_cb = ?*const fn (?*anyopaque, ares_socket_t, c_int, c_int) callconv(.C) void;
 pub const struct_apattern = opaque {};
 const fd_set = c.fd_set;
+
+pub const NSClass = enum(c_int) {
+    ns_c_invalid = 0, // Cookie.
+    ns_c_in  = 1, // Internet.
+    ns_c_2  = 2, // unallocated/unsupported.
+    ns_c_chaos  = 3, // MIT Chaos-net.
+    ns_c_hs  = 4, // MIT Hesiod.
+    // Query class values which do not appear in resource records
+    ns_c_none = 254, // for prereq. sections in update requests
+    ns_c_any = 255, // Wildcard match.
+    ns_c_max = 65536,
+};
+
+pub const NSType = enum(c_int) {
+    ns_t_invalid = 0, // Cookie.
+    ns_t_a = 1, // Host address.
+    ns_t_ns = 2, // Authoritative server.
+    ns_t_md = 3, // Mail destination.
+    ns_t_mf = 4, // Mail forwarder.
+    ns_t_cname = 5, // Canonical name.
+    ns_t_soa = 6, // Start of authority zone.
+    ns_t_mb = 7, // Mailbox domain name.
+    ns_t_mg = 8, // Mail group member.
+    ns_t_mr = 9, // Mail rename name.
+    ns_t_null = 10, // Null resource record.
+    ns_t_wks = 11, // Well known service.
+    ns_t_ptr = 12, // Domain name pointer.
+    ns_t_hinfo = 13, // Host information.
+    ns_t_minfo = 14, // Mailbox information.
+    ns_t_mx = 15, // Mail routing information.
+    ns_t_txt = 16, // Text strings.
+    ns_t_rp = 17, // Responsible person.
+    ns_t_afsdb = 18, // AFS cell database.
+    ns_t_x25 = 19, // X_25 calling address.
+    ns_t_isdn = 20, // ISDN calling address.
+    ns_t_rt = 21, // Router.
+    ns_t_nsap = 22, // NSAP address.
+    ns_t_nsap_ptr = 23, // Reverse NSAP lookup (deprecated).
+    ns_t_sig = 24, // Security signature.
+    ns_t_key = 25, // Security key.
+    ns_t_px = 26, // X.400 mail mapping.
+    ns_t_gpos = 27, // Geographical position (withdrawn).
+    ns_t_aaaa = 28, // Ip6 Address.
+    ns_t_loc = 29, // Location Information.
```

Comment:
if you use `///` and put it above, it will appear in autocomplete

Reply:
done